wolox-training/igc-rails
# Blog

### Summary
- Implemented all features detailed for the tutorial blog project in http://guides.rubyonrails.org/getting_started.html.
- Changed sqlite3 to postgresql, and erb to slim.

### Trello Card
https://trello.com/c/n1LLvNdk/23-crear-blog-base

### Test Plan
(when prompted, username is "dhh", password is "secret"
1. Go to /articles
2. Create a new article. It should prompt for credentials.
3. Add new comments. 
4. Comments should be visible on the article page.
5. Destroy a comment. It should display a confirm window and then prompt for credentials (inputting wrong credentials should prevent the comment from being deleted).
6. Go back to /articles.
7. Destroy the article. It should display a confirm window and then prompt for credentials.
8. Comment should no longer appear on the listing page.
